front-end/src/components/cookie/cookie-view.module.scss

```diff
@@ -0,0 +1,105 @@
+/*** 
+    Cookie view uses Bootstrap modal to display the view. As such, it uses some style overrides 
+    that come from the Bootstrap css.
+***/
+
+@import "src/assets/stylesheets/fonts";
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/globals";
+
+// Cookie view related constants
+$cookie-view-font-size: 1.4rem;
+$cookie-body-letter-spacing: -0.05rem;
+$cookie-close-button-font-size: 3.2rem;
+
+$color-cookie-view-bg: rgba(228, 239, 248, 1);
+
+.cookie-modal-backdrop {
+    @extend .common-fixed-position-rules;
+
+    z-index: map-get($map: $z-index, $key: overlay);
+    background-color: $color-primary;
+    opacity: 0.7;
+}
+
+.cookie-modal {
+    @extend .common-fixed-position-rules;
+
+    z-index: map-get($map: $z-index, $key: modal);
+    background-color: $color-cookie-view-bg;
+    
+    height: 19rem;
+    font-size: $cookie-view-font-size;
+    margin: 12em 1em 0 1em;
+    padding: 2rem;
+
+    :global(.modal-content) {
+        background-color: transparent;
+        border-radius: 0;
+        box-shadow: none;
+        border: 0;
+    }
+
+    :global(.modal-header) {
+        @extend .font-l;
+
+        padding: 0;
+        border-bottom: none;
```

Comment:
If possible please have specific-first basis of styling.

That is, for example, if you have a tree of nodes 1 or 2 elements having border and the others dont, then set the border style specific to the 1 or 2 elements that have it, instead of setting it globally and override everywhere

Reply:
Completely true. This however comes from bootstrap css, which sets the borders. Which is why we override it here.

Reply:
Completely true. This however comes from bootstrap css, which sets the borders. You can see the .modal-header has a global modifier. Which is why we override it here.

Reply:
ah, i see